style.css

```diff
@@ -0,0 +1,177 @@
+
+h1 {
+    font-size: 56px;
+    font-family: 'Segoe UI';
+    text-transform: uppercase;
+    text-align: center;
+    line-height: 1;
+}
+
+h2 {
+    font-size: 18px;
+    font-family: 'Segoe UI';
+}
+
+h3 {
+    font-size: 24px;
+    font-family: Tahoma;
+    text-transform: uppercase;
+    color: #ffffff;
+}
+
+.block-title {
+    background-color: rgb(245, 0, 102);
+}
+
+mark {
+    font-size: 20px;
+    font-family: 'Segoe UI';
+    color: #ffffff;
+    background-color: rgb(235, 64, 135);
+    text-transform: uppercase;
+    margin: 10px;
+    padding: 4px;
+}
+
+p {
+    font-family: 'Times New Roman';
+    font-size: 16px;
+    line-height: 2;
+}
+
+i {
+    font-family: serif;
+    font-size: 24px;
+    display: block;
+    text-align: center;
+    font-style: normal;
+    
+}
+
+.title-1 {
+    font-weight: bold;
+    text-transform: uppercase;
+    color: rgb(235, 64, 135);
+    
+}
+
+cite {
+    font-size: 12px;
+    font-style: normal;
+    display: block;
+    text-align: center;
+    border-bottom: 1px dotted;
+    margin-top: 10px;
+}
+
+strong {
+    color: rgb(235, 64, 135);
+    font-weight: bold;
+    font-size: 20px;
+    background-color: rgb(0, 0, 0);
+    padding: 5px 10px;
+    margin-right: 5px;
+}
+
+.highlighted-words {
+    font-family: 'Trebuchet MS';
+}
+
+.figcaption {
+    font-size: 12px;
+}
+
+li {
+    list-style-type: square;
+    color: rgb(221, 0, 92);
+
+}
+
+li span {
+    color: rgb(0, 0, 0);
+    line-height: 2;
+}
+
+aside {
+    background-color: rgb(0, 0, 0);
+    width: 260px;
+    height: 400px;
+}
+
+.links-to-read {
+    font-size: 14px;
+    font-family: Tahoma;
+    color: #ffffff;
+}
+
+a {
+    font-size: 12px;
+    font-weight: bold;
+    font-family: Tahoma;
+    text-decoration: none;
+    color: #ffffff;
+}
+
+.quotes {
+    background: rgb(0, 0, 0);
+    color: #ffffff;
+    padding: 3px 8px;
+}
+
+blockquote {
+    text-transform: uppercase;
+}
+
+.rectangle {
+    width: 10px;
+    height: 30px;
+    margin-left: 12px;
+    background-color: rgb(235, 64, 135);
+}
+
+.page-one {
```

Comment:
Again the same comment. 
1) We don't call smth like 'xxx-first', 'xxx-one', 'xxx-1', etc. Try to describe what this is about, not how it looks like or where id is placed.
2) It's not a pdf, we have just one page and all the content will be on one page, so the name is confusing

![image](https://user-images.githubusercontent.com/104353194/169477712-e5082b00-ff91-4289-b6cd-970f8a6af15f.png)